src/main/java/com/gendeathrow/morechickens/modHelper/MoreRefinedStorageAddon.java

```diff
@@ -0,0 +1,41 @@
+package com.gendeathrow.morechickens.modHelper;
+
+import java.util.List;
+
+import com.setycz.chickens.handler.SpawnType;
+import com.setycz.chickens.registry.ChickensRegistryItem;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+public class MoreRefinedStorageAddon extends BaseModAddon
+{
+
+	public static ChickensRegistryItem crystalizedChicken = null;
+
+	public MoreRefinedStorageAddon()
+	{
+		super("morerefinedstorage", "More Refined Storage", "textures/entity/morerefinedstorage/");
+		this.setNeedsModPresent(false);
```

Comment:
See my other comment.

Unless there are other mods that add this `Crystalized Nugget`, then you've gone about this wrong